src/universal/components/TeamInvitationGoogleCreateAccount.tsx

```diff
@@ -140,6 +144,7 @@ export default createFragmentContainer(
   withAtmosphere(withMutationProps(withRouter(TeamInvitationGoogleCreateAccount))),
   graphql`
     fragment TeamInvitationGoogleCreateAccount_verifiedInvitation on VerifiedInvitationPayload {
+      meetingType
```

Comment:
@mattkrick I’m stumped as to why `meetingType` doesn’t come through in the payload. This is the only hint:
```
index.js:3740 Warning: RelayResponseNormalizer(): Payload did not contain a value for field `meetingType: meetingType`. Check that you are parsing with the same query that was used to fetch the payload.
```

Reply:
likely a stale fragment on the server. try restarting the server, if that doesn't work, maybe running `yarn postdeploy`?